### DISCUSSION TITLE
Need plugin to display a map point TO my current location.

### DISCUSSION THREAD

**Discussion by jackhen** [osmandapp/OsmAnd#15512] [Maps Related]
I need to tell my group what my position is on a map. I have predefined favorites on this map already marked. So say if I am closest to Grid point 1, I want to tell my group over radio three things so they can mark my position with their map.
1. Grid point name
2.My bearing position relative to this grid point.
3.My distance from the grid point.
Now my team knows where I am on the map. The only issue is #2. Using markers and the compass overlay I can show my position TO the grid point. However I need to know the bearing at the grid point TO my location.
![Bearing](https://user-images.githubusercontent.com/22885374/196767865-e74d8e36-0b3d-40be-83b7-99cbc7a74ded.png)

**Comment by vshcherb** [comment]
There is a bearing to Marker if you enable it, so in that case you will need to update marker to a target position.  IF that's a moving position, probably OsmAnd Online Tracker plugin could help

**Reply by jackhen** [reply]
I know that. But as I mentioned, I need a Marker bearing to position if you will.

**Reply by tommycrock** [reply]
Is there something I'm misunderstanding. Is it not just the opposite? E.g. in degrees just add/subtract 180?

**Comment by jackhen** [comment]
Exactly! So I am asking for a plugin that does this exact thing. OR, enhance the bearing plugin with a 4th choice. Either way I don't want to do the math. I want the readout immediately from a plugin.
When I have 15 team members calling in their locations, "doing the math" doesn't cut it.
